connexion/lifecycle.py

```diff
@@ -21,6 +20,7 @@ def __init__(self,
         self.json_getter = json_getter
         self.files = files
         self.context = context if context is not None else {}
+        self.content_type = self.headers.get("Content-Type", "")
```

Comment:
If that's a derived thingy, perhaps a property and returning `None` if a content-type not set?

Reply:
:+1: